### ISSUE TITLE
Implement a `minikube disk-usage` command so users can see how much space minikube is using

### ISSUE THREAD

**Issue by spowelljr** [kubernetes/minikube#12967]
There have been some comments by users that they find minikube uses a lot of disk space and there's no easy way to see how much it's using.

We should implement a command so users can easily view the disk space usage. This will include things like preloads and image caching and output the usage categorically.

**Comment by afbjorklund** [comment]
Related to:
* #3574

**Comment by afbjorklund** [comment]
> There have been some comments by users that they find minikube uses a lot of disk space and there's no easy way to see how much it's using.

This is also relative.

Minikube does consume a **lot** of disk space, compared to previous solutions like docker-machine and localkube.

But it should consume a _comparable_ amount, when compared to something like vagrant and plain kubeadm ?

A couple of GiB is "expected".

**Comment by medyagh** [comment]
I agree with the fucntionatliy be useful,  how about adding a generic command called
that could later add more stuff in addition to disk usage.


"minikube info"

**Comment by medyagh** [comment]
this item is available and I would accept a Pr
